helm-charts/UpgradeManager/templates/rbac.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fum
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
I thought Role and Rolebinding should be enough?  Also this chart will be applied by KubeFATE, who only has Role and Rolebinding edit permissions.